molo/core/tests/test_cas.py

```diff
@@ -84,7 +84,7 @@ def test_succesful_login(self, mock_verify):
         mock_verify.return_value = (
             'test@example.com',
             {'ticket': 'fake-ticket', 'service': service, 'has_perm': 'True',
-             'is_admin': 'True'},
+             'is_admin': 'True', 'email': 'test@example.com'},
```

Comment:
This works, but I'd also like to get another test that checks that the user that's created has the email field set.